roles/azure_load_balancer_with_public_ip/tasks/delete.yml

```diff
@@ -21,3 +21,16 @@
     resource_group: "{{ azure_load_balancer_with_public_ip_resource_group }}"
     name: "{{ pip.response[0].name }}"
     state: absent
+
+- name: Wait 120 seconds before deleting resource group
```

Comment:
Added this wait because it takes time for the load balancer to get actual deleted
Any other suggestions here?

Reply:
This was solved by adding retries to the Resource Group delete.
See PR for more info:
https://github.com/redhat-cop/cloud.azure_ops/pull/74